OpenRA.Mods.Common/UpdateRules/Rules/20210321/Infiltrates.cs

```diff
@@ -0,0 +1,29 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2022 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class Infiltrates : UpdateRule
+	{
+		public override string Name => "Removes PlayerExperience property from Infiltrates.";
+
+		public override string Description => "Removes PlayerExperience property from Infiltrates and disperses it to all InfiltrateFor Traits.";
+
+		public override IEnumerable<string> UpdateActorNode(ModData modData, MiniYamlNode actorNode)
+		{
+			actorNode.ChildrenMatching("Infiltrates").Do(i => i.RemoveNodes("PlayerExperience"));
```

Comment:
This should probably mention that manual edits are required after a `PlayerExperience` node was removed. See `RemovePlaceBuildingPalette` as example.

Reply:
Updated it to be similar